addons/br/nfse/line.go

```diff
@@ -0,0 +1,18 @@
+package nfse
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/regimes/br"
+	"github.com/invopop/validation"
+)
+
+func validateLine(value any) error {
+	line, _ := value.(*bill.Line)
+	if line == nil {
+		return nil
+	}
+
+	return validation.Validate(line,
+		bill.RequireLineTaxCategory(br.TaxCategoryISS),
```

Comment:
Does this also need to be repeated for Discount and Charge and lines?

Reply:
Actually, I don't think it makes sense to support either Charges or Discounts: it doesn't seem possible to map them to an NFSe. So, I've added validations to ensure those aren't present in [177eb69](https://github.com/invopop/gobl/pull/402/commits/177eb697a8a4bf0fecd5edf6bc1cab717d7fa216)